python/GafferUI/EventLoop.py

```diff
@@ -198,6 +198,25 @@ def removeIdleCallback( cls, callback ) :
 
 		cls.__idleCallbacks.remove( callback )
 
+	## Convenience method to introduce a delay on the mainEventLoop().
+	@classmethod
+	def waitForIdle( cls, count = 1000 ) :
```

Comment:
I still don't like putting this front and centre in EventLoop - it is just a hack for the screen grab app (and the tests) after all. Given that the tests are acting a bit wacky on OSX (but not in a way that would actually harm our desired usage), I suggest we either just whack it on the app or you use the one from GafferUITest....